sphinx/texinputs/sphinx.sty

```diff
@@ -898,6 +899,7 @@
 \DisableKeyvalOption{sphinx}{numfigreset}
 \DisableKeyvalOption{sphinx}{nonumfigreset}
 \DisableKeyvalOption{sphinx}{mathnumfig}
+\DisableKeyvalOption{sphinx}{mathnumsep}
```

Comment:
Any reason to disable usage of this key via raw injection of `\sphinxsetup`?

Reply:
I do not see a strong reason to disable this key after sphinx loading time, but I love to hack LaTeX so the perspective to change the separator locally was appealing to me, but let's go with it as I must admit not being sole Sphinx user in the world.

Reply:
And actually, I would like to be able to hack `numfigreset` in `\sphinxsetup` because of the different ways we number HTML and LaTeX documents.

Reply:
ok, if you want to be able to use distinct settings the easiest (barring an official interface with an added specific to LaTeX numsep config variable) is then to not use this `\DisableKeyvalOption` line.

+1 for removing it.

Reply:
```suggestion
```